software/design/milestone_v0.2.md

```diff
@@ -45,7 +45,6 @@ Specific tasks are captured as GitHub Issues in the following milestones:
  - [v0.2 GUI](https://github.com/RespiraWorks/Ventilator/milestone/12)
  - [v0.2 Alarms](https://github.com/RespiraWorks/Ventilator/milestone/11)
  - [Pressure Assist Mode](https://github.com/RespiraWorks/Ventilator/milestone/8)
- - [HFNC mode](https://github.com/RespiraWorks/Ventilator/milestone/9)
```

Comment:
Why is this being removed?

Reply:
> Why is this being removed?

The link is flagged as dead. I think this comes from the cleaning up of milestones.

Reply:
Ah yes, we did delete that milestone, as it was empty. As this is a historical document, I'd maybe leave it in the list but without the link? But this is trivial, so whatever.